doc/spec_exceptionHandling.dox

```diff
@@ -1,20 +1,21 @@
 // put the namespace around the doxygen block so we don't have to give it all the time in the code to get links
 namespace ChimeraTK {
 /**
-\page spec_execptionHandling Technical specification: Exception handling for device runtime errors V1.0
-
-> This version is identical to V1.0RC2WIP.
+\page spec_execptionHandling Technical specification: Exception handling for device runtime errors V1.1RC2WIP
 
 > **NOTICE FOR FUTURE RELEASES: AVOID CHANGING THE NUMBERING!** The tests refer to the sections, incl. links and unlinked references from tests or other parts of the specification. These break, or even worse become wrong, when they are not changed consistenty!
 
 \section spec_execptionHandling_intro A. Introduction
 
 - 1. Exceptions are handled by ApplicationCore in a way that the application developer does not need to care much about it.
 
-- 2. ChimeraTK::runtime_error exceptions are caught by the framework and are reported to the DeviceModule.
-- 3. The DeviceModule handles this exception and periodically tries to reopen the device.
+- 2. ChimeraTK::runtime_error exceptions are caught by the framework and are reported to the DeviceManager.
+- 3. The DeviceManager handles this exception and periodically tries to reopen the device.
 - 4. Communication with the faulty device is _skipped_, _frozen_ or _delayed_ until the device is functional again (see \ref spec_exceptionHandling_intro_terminology "A.9").
 - 5. In case of several devices only the faulty device is affected.
```

Comment:
AS discussed, remove 5 and promote 5.1 to 5 with children 5.2 -> 5.1 and 5.3 -> 5.2